product-service/mockData/products.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+module.exports.products = [
```

Comment:
NP: You could use only mock json file here, not module

Reply:
Good catch, I will update my PR. Thanks